src/components/atoms/Search/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import SearchIcon from '@mui/icons-material/Search';
+import IconButton from '@mui/material/IconButton/IconButton';
+import InputBase from '@mui/material/InputBase';
+
+const style = {
+    background: 'linear-gradient(45deg, #f2f2f2 30%, #f2f2f2 90%)',
+    borderRadius: 25,
+    color: 'black'
+};
+
+const Search = ({ onSearch }: { onSearch: (term: string) => void }) => {
+    const [searchTerm, setSearchTerm] = useState('');
+
+    const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        const value = e.target.value;
+        setSearchTerm(value);
+        onSearch(value); // Aquí utilizamos onSearch
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (performance):** Invoking `onSearch` directly within `handleChange` will trigger the search operation on every keystroke, which might not be efficient, especially for API-driven searches. Consider debouncing the search operation to limit the number of requests made while typing.